### PR TITLE
MAGETWO-95591 patch

### PR DESCRIPTION
**Pull request by jfrontain**
## This PR is a:

- [ ] New topic
- [ x] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will...

## Additional information

List all affected URLs 

- ...
- ...

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=3191) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=3191) before we can accept your contribution.<br/><hr/>**Jeanne Frontain** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

File: guides/v2.2/release-notes/ReleaseNotes2.2.6EE.md [hunk 1]
```diff
@@ -1744,15 +1744,16 @@ You can find Magento Shipping-specific release notes in [Magento Shipping Releas
...
 
-* Remove   `pub/media/catalog/product/cache`. (The path for cached images was changed in this release, which explains why you need to clean this directory after upgrade to free up space.) 
 
-* Run `bin/magento catalog:image:resize`  to generate a new image cache.  (This step is necessary because we’ve changed the path to cached images and must remove the previously cached images.)
+**Issue**: The `catalog:image:resize` command execution time has been reduced by up to 90% in the release. However, this improvement necessitates these additional steps after upgrading your Magento instance to 2.2.6: 
 
+	* Remove   `pub/media/catalog/product/cache`. (The path for cached images was changed in this release, which explains why you need to clean this directory after upgrade to free up space.)
```
Comment (lorikrell) [inline guides/v2.2/release-notes/ReleaseNotes2.2.6EE.md]: The info in ( ) is pretty important. I'd remove the ( ) and leave the sentences as is. The info is really helpful.

File: guides/v2.2/release-notes/ReleaseNotes2.2.6EE.md [hunk 2]
```diff
@@ -1744,15 +1744,16 @@ You can find Magento Shipping-specific release notes in [Magento Shipping Releas
...
 
-* Remove   `pub/media/catalog/product/cache`. (The path for cached images was changed in this release, which explains why you need to clean this directory after upgrade to free up space.) 
 
-* Run `bin/magento catalog:image:resize`  to generate a new image cache.  (This step is necessary because we’ve changed the path to cached images and must remove the previously cached images.)
+**Issue**: The `catalog:image:resize` command execution time has been reduced by up to 90% in the release. However, this improvement necessitates these additional steps after upgrading your Magento instance to 2.2.6: 
 
+	* Remove   `pub/media/catalog/product/cache`. (The path for cached images was changed in this release, which explains why you need to clean this directory after upgrade to free up space.) 
 
+	* Run `bin/magento catalog:image:resize`  to generate a new image cache.  (This step is necessary because we’ve changed the path to cached images and must remove the previously cached images.)
```
Comment (lorikrell) [inline guides/v2.2/release-notes/ReleaseNotes2.2.6EE.md]: Same about the ( ), remove them as the info is very important, no need to make it an aside.

File: guides/v2.2/release-notes/ReleaseNotes2.2.6CE.md [hunk 3]
```diff
@@ -1677,13 +1677,16 @@ You can find Magento Shipping-specific release notes in [Magento Shipping Releas
...
 
-* Remove   `pub/media/catalog/product/cache`. (The path for cached images was changed in this release, which explains why you need to clean this directory after upgrade to free up space.) 
 
-* Run `bin/magento catalog:image:resize`  to generate a new image cache.  (This step is necessary because we’ve changed the path to cached images and must remove the previously cached images.)
+**Issue**: The `catalog:image:resize` command execution time has been reduced by up to 90% in the release. However, this improvement necessitates these additional steps after upgrading your Magento instance to 2.2.6: 
+
+	* Remove   `pub/media/catalog/product/cache`. (The path for cached images was changed in this release, which explains why you need to clean this directory after upgrade to free up space.)
```
Comment (lorikrell) [inline guides/v2.2/release-notes/ReleaseNotes2.2.6CE.md]: I'd remove the ( ), info is good and needed, no need for aside.

File: guides/v2.2/release-notes/ReleaseNotes2.2.6CE.md [hunk 4]
```diff
@@ -1677,13 +1677,16 @@ You can find Magento Shipping-specific release notes in [Magento Shipping Releas
...
 
-* Remove   `pub/media/catalog/product/cache`. (The path for cached images was changed in this release, which explains why you need to clean this directory after upgrade to free up space.) 
 
-* Run `bin/magento catalog:image:resize`  to generate a new image cache.  (This step is necessary because we’ve changed the path to cached images and must remove the previously cached images.)
+**Issue**: The `catalog:image:resize` command execution time has been reduced by up to 90% in the release. However, this improvement necessitates these additional steps after upgrading your Magento instance to 2.2.6: 
+
+	* Remove   `pub/media/catalog/product/cache`. (The path for cached images was changed in this release, which explains why you need to clean this directory after upgrade to free up space.) 
+
+	* Run `bin/magento catalog:image:resize`  to generate a new image cache.  (This step is necessary because we’ve changed the path to cached images and must remove the previously cached images.)
```
Comment (lorikrell) [inline guides/v2.2/release-notes/ReleaseNotes2.2.6CE.md]: I'd remove the ( ), info is good and needed, no need for aside.

File: guides/v2.2/release-notes/ReleaseNotes2.2.6CE.md [hunk 5]
```diff
@@ -1677,13 +1677,16 @@ You can find Magento Shipping-specific release notes in [Magento Shipping Releas
 
 
 
-## Known issue
+## Known issues
 
-The `catalog:image:resize` command execution time has been reduced by up to 90% in the release. However, this improvement necessitates these additional steps after upgrading your Magento instance to 2.2.6: 
+**Issue**: Merchants cannot save a newly created multiselect or dropdown customer attribute or edit existing customer attributes from the customer’s account on the storefront after upgrade to Magento 2.2.6. **Workaround**: Download and apply hot fix [MAGETWO-95591](https://magento.com/tech-resources/download#download2248). This issue will be fixed in Magento Open Source 2.2.7, which is scheduled for release by the end of 2018.
```
Comment (lorikrell) [inline guides/v2.2/release-notes/ReleaseNotes2.2.6CE.md]: Should the workaround be on it's own line, to highlight the workaround? I like the added issue and workaround, the info as to the issue and fix, etc.

File: guides/v2.2/release-notes/ReleaseNotes2.2.6EE.md [hunk 6]
```diff
@@ -1744,15 +1744,16 @@ You can find Magento Shipping-specific release notes in [Magento Shipping Releas
 
 
 
-## Known issue
+## Known issues
 
-The `catalog:image:resize` command execution time has been reduced by up to 90% in the release. However, this improvement necessitates these additional steps after upgrading your Magento instance to 2.2.6: 
+**Issue**: Merchants cannot save a newly created multiselect or dropdown customer attribute or edit existing customer attributes from the customer’s account on the storefront after upgrade to Magento 2.2.6. **Workaround**: Download and apply  Magento Commerce Support Patch MAGETWO-95591 after logging into your account from the partner portal. This issue will be fixed in Magento Commerce 2.2.7, which is scheduled for release by the end of 2018.
```
Comment (lorikrell) [inline guides/v2.2/release-notes/ReleaseNotes2.2.6EE.md]: Should the workaround be on it's own line, to highlight the workaround? I like the added issue and workaround, the info as to the issue and fix, etc.

**Pull request merged**